cookbook/docs/getting_started/flyte_fundamentals.md

```diff
@@ -0,0 +1,55 @@
+(getting_started_fundamentals)=
+
+# Flyte Fundamentals
+
+This section of the **Getting Started** documentation will take you through the
+fundamental concepts of Flyte: tasks, workflows, and launch plans.
+
+You'll learn about the full development lifecycle of creating a project,
+registering workflows, and running them on a demo Flyte cluster. These
+guides will also walk you through how to visualize artifacts associated with
+tasks, optimize them for scale and performance, and extend Flyte for your own
+use cases.
+
+```{list-table}
+:header-rows: 0
+:widths: 20 30
+
+* - {doc}`🔀 Tasks, Workflows and LaunchPlans <tasks_and_workflows>`
```

Comment:
Can we also include emojis in the left nav?

Reply:
I tried it out and it looked a little too busy